packages/tenants/tests/tenants-askar-profiles.e2e.test.ts

```diff
@@ -0,0 +1,128 @@
+import type { InitConfig } from '@aries-framework/core'
+
+import { Agent } from '@aries-framework/core'
+import { agentDependencies } from '@aries-framework/node'
+
+import { describeRunInNodeVersion } from '../../../tests/runInVersion'
+import { AskarModule, AskarMultiWalletDatabaseScheme, AskarProfileWallet, AskarWallet } from '../../askar/src'
+import { askarModuleConfig } from '../../askar/tests/helpers'
+import { testLogger } from '../../core/tests'
+
+import { TenantsModule } from '@aries-framework/tenants'
+
+// FIXME: Re-include in tests when Askar NodeJS wrapper performance is improved
```

Comment:
This can be removed, right?

Reply:
Not really. The tests still run only in Node 18. It can be removed when #1510 is resolved

Reply:
Maybe the comment is a bit outdated because it was done when tests were not running in CI at all. Some time after that, the "describeRunIf" trick has been added in order to run them only on node 18.

But we can live with these comments for now... #1510 will be resolved very soon! 😄